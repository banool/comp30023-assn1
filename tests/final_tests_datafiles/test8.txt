0 1 150 60
1 2 2 2
15 3 150 3
17 4 4 4
18 5 1 2
19 6 1 1
30 7 1 60
31 8 150 3
32 9 100 2
33 10 2 3
