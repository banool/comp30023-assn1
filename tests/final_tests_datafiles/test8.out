time 0, 1 running, numprocesses=1, numholes=1, memusage=94%
time 2, 2 running, numprocesses=2, numholes=1, memusage=95%
time 4, 1 running, numprocesses=1, numholes=1, memusage=94%
time 8, 1 running, numprocesses=1, numholes=1, memusage=94%
time 16, 3 running, numprocesses=1, numholes=1, memusage=94%
time 18, 4 running, numprocesses=2, numholes=1, memusage=97%
time 20, 5 running, numprocesses=3, numholes=1, memusage=97%
time 22, 6 running, numprocesses=3, numholes=1, memusage=97%
time 23, 3 running, numprocesses=2, numholes=1, memusage=97%
time 24, 4 running, numprocesses=1, numholes=2, memusage=3%
time 26, 1 running, numprocesses=1, numholes=1, memusage=94%
time 34, 7 running, numprocesses=2, numholes=1, memusage=95%
time 36, 8 running, numprocesses=2, numholes=1, memusage=95%
time 38, 9 running, numprocesses=2, numholes=2, memusage=64%
time 40, 10 running, numprocesses=2, numholes=2, memusage=2%
time 42, 7 running, numprocesses=2, numholes=2, memusage=2%
time 46, 8 running, numprocesses=2, numholes=1, memusage=95%
time 47, 10 running, numprocesses=2, numholes=2, memusage=2%
time 48, 1 running, numprocesses=2, numholes=1, memusage=95%
time 56, 7 running, numprocesses=2, numholes=1, memusage=95%
time 64, 1 running, numprocesses=2, numholes=1, memusage=95%
time 72, 7 running, numprocesses=2, numholes=1, memusage=95%
time 80, 1 running, numprocesses=2, numholes=1, memusage=95%
time 88, 7 running, numprocesses=2, numholes=1, memusage=95%
time 96, 1 running, numprocesses=2, numholes=1, memusage=95%
time 104, 7 running, numprocesses=2, numholes=1, memusage=95%
time 112, 1 running, numprocesses=2, numholes=1, memusage=95%
time 118, 7 running, numprocesses=1, numholes=2, memusage=1%
time 126, 7 running, numprocesses=1, numholes=2, memusage=1%
time 134, 7 running, numprocesses=1, numholes=2, memusage=1%
time 140, simulation finished.
